src/tests/automated_tests/unit/test_unit_utilities.py

```diff
@@ -27,40 +27,45 @@
 
 
 # test data for rename, chmod,chown, download,upload
-DATA = [ (SERVER_UTILITIES, 200) , ("someservernotavailable", 400)]  
+DATA = [ (SERVER_UTILITIES, 200) , ("someservernotavailable", 400)]
 
 # test data for file
-DATA_FILE = [ (SERVER_UTILITIES, 200, ".bashrc") , 
-         ("someservernotavailable", 400, ".bashrc"), 
-		 (SERVER_UTILITIES, 400, "nofile") , 
-		 (SERVER_UTILITIES, 400, "/var/log/messages") , 
-		 ]  
+DATA_FILE = [ (SERVER_UTILITIES, 200, ".bashrc") ,
+         ("someservernotavailable", 400, ".bashrc"),
+		 (SERVER_UTILITIES, 400, "nofile") ,
+		 (SERVER_UTILITIES, 400, "/var/log/messages") ,
+		 (SERVER_UTILITIES, 400, "/\\") ,
+		 (SERVER_UTILITIES, 400, "d[") ]
 
 # test data for #mkdir, symlink
-DATA_201 = [ (SERVER_UTILITIES, 201) , ("someservernotavailable", 400)]  
+DATA_201 = [ (SERVER_UTILITIES, 201) , ("someservernotavailable", 400)]
 
 # test data for ls command
-DATA_LS = [ (SERVER_UTILITIES, "/srv/f7t/test_sbatch.sh", 200), 
+DATA_LS = [ (SERVER_UTILITIES, "/srv/f7t/test_sbatch.sh", 200),
 (SERVER_UTILITIES, USER_HOME + "/dontexist/", 400),
 (SERVER_UTILITIES, "/etc/binfmt.d/", 200), # empty folder
 (SERVER_UTILITIES, USER_HOME + "/", 200),
-("someservernotavailable", USER_HOME + "/" ,400)]  
+("someservernotavailable", USER_HOME + "/" ,400)]
+
+FORBIDDEN_INPUT_CHARS = '<>|;"\'&\\[]()\x00\x0D\x1F'
+for c in FORBIDDEN_INPUT_CHARS:
+    DATA_LS.append((SERVER_UTILITIES, "/bin/*" + c, 400))
 
 # test data for checksum API
-DATA_CK = [ (SERVER_UTILITIES, "/srv/f7t/test_sbatch.sh", 200), 
-(SERVER_UTILITIES, "/srv/f7t/test_sbatch_forbidden.sh", 400), 
+DATA_CK = [ (SERVER_UTILITIES, "/srv/f7t/test_sbatch.sh", 200),
+(SERVER_UTILITIES, "/srv/f7t/test_sbatch_forbidden.sh", 400),
 (SERVER_UTILITIES, USER_HOME + "/dontexist/", 400),
 (SERVER_UTILITIES, "/etc/binfmt.d/", 400), # empty folder
 (SERVER_UTILITIES, USER_HOME + "/", 400),
-("someservernotavailable", USER_HOME + "/" ,400)]  
+("someservernotavailable", USER_HOME + "/" ,400)]
 
 # test data for checksum API
-DATA_VIEW = [ (SERVER_UTILITIES, "/srv/f7t/test_sbatch.sh", 200), 
+DATA_VIEW = [ (SERVER_UTILITIES, "/srv/f7t/test_sbatch.sh", 200),
 (SERVER_UTILITIES, "/bin/wc", 400), # non ASCII file
 (SERVER_UTILITIES, "/lib64/libz.so", 400), # non ASCII file
-(SERVER_UTILITIES, "/srv/f7t/test_sbatch_forbidden.sh", 400), 
+(SERVER_UTILITIES, "/srv/f7t/test_sbatch_forbidden.sh", 400),
 (SERVER_UTILITIES, USER_HOME + "/dontexist/", 400),
-(SERVER_UTILITIES, "/slurm-19.05.4.tar.bz2", 400), # > MAX_SIZE file
+(SERVER_UTILITIES, "/slurm-20.11.7.tar.bz2", 400), # > MAX_SIZE file
```

Comment:
I think that it'd be good if we could copy or move the slurm tarball into a more generic name (ie: slurm.tar.bz2), not depending on the version, in order to not change the tests each time it is updated (even when version change would only happen once in a while)